src/main/java/springboot/webapp/usersmanager/entities/User.java

```diff
@@ -0,0 +1,40 @@
+package springboot.webapp.usersmanager.entities;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import lombok.Setter;
+
+import javax.persistence.*;
+
+@Entity
+@Setter
+@Getter
+@NoArgsConstructor
+@Table(name = "users")
+public class User {
```

Comment:
Which field is unique for your user? How will you handle with authorization in future?